### ISSUE TITLE
Missing lang in html element

### ISSUE THREAD

**Issue by cswitenky** [cswitenky/cswitenky#4]
If a page doesn't specify a `lang` attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly. [Learn more about the lang attribute](https://dequeuniversity.com/rules/axe/4.8/html-has-lang).

Source: [PageSpeed Insights](https://pagespeed.web.dev/analysis/https-switenky-com/orn9rwxga7)
